terraform/modules/heimdall/variables.tf

```diff
@@ -44,3 +39,8 @@ variable "cert" {
   type        = string
   description = "Local path to the TLS certificate"
 }
+
+variable "karen_endpoint" {
```

Comment:
I think this should be like a `hosts` or `endpoints` variable that's a map of the endpoints to the other services and inside the `task_definition` block, you can use [directives](https://www.terraform.io/docs/configuration/expressions.html#directives) to fill out the environment variables.

Reply:
Okay, I'll change it.